### PR TITLE
Merge and upgrade voting module 

### PR DESCRIPTION
**Pull request by MSNTCS**
In this PR, we merge the voting module to the new branch, voting, and upgrade its dependencies to the latest version.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -54,6 +54,7 @@ tokio-core = "0.1.17"
 toml = "0.4"
```
Comment (majecty) [inline Cargo.toml:54]: The one who read this commit message will not know what is merged.
How about "Implement voting application"?

Comment (majecty) [reply Cargo.toml:54]: Is the "Merge Voting module" commit the same as the previous PRs?
If you did not change any code, I'll not check the code in detail.

Please tell me if you changed something.

File: voting/src/common.rs [hunk 2]
```diff
@@ -48,7 +48,7 @@ pub struct SignedTransaction<T: Action> {
 impl<T: Action> SignedTransaction<T> {
     pub fn verify(&self) -> Result<(), ()> {
         let message = self.tx.hash();
-        if verify(&self.signature, &message, &self.signer_public) {
+        if verify(&self.signature, &message.0.as_ref(), &self.signer_public) {
```
Comment (majecty) [inline voting/src/common.rs:51]:
```suggestion
        if verify(&self.signature, &message.as_ref(), &self.signer_public) {
```

File: voting/Cargo.toml [hunk 3]
```diff
@@ -5,14 +5,13 @@ authors = ["CodeChain Team <hi@codechain.io>"]
...
-ccrypto = { package = "codechain-crypto", git = "https://github.com/CodeChain-io/rust-codechain-crypto.git", version = "0.2" }
+ccrypto = { package = "codechain-crypto", git = "https://github.com/CodeChain-io/rust-codechain-crypto.git", version = "0.3" }
 ckey = {package = "codechain-key", path = "../key"}
 coordinator = { path = "../coordinator" }
-foundry-module-rt = { version = "0.1.1", git = "https://github.com/CodeChain-io/module-rt.git", tag = "v0.1.1"}
+foundry-module-rt = { version = "0.2.0", git = "https://github.com/CodeChain-io/module-rt.git", tag = "v0.2.0" }
 linkme = "0.2.1"
 parking_lot = "0.11.0"
-primitives = { git = "https://github.com/CodeChain-io/rust-codechain-primitives.git", version = "0.4" }
-rand = "0.6.1"
```
Comment (majecty) [inline voting/Cargo.toml]: :+1:

**Review by majecty — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (MSNTCS) [reply Cargo.toml:54]: @majecty it is the same as previous PRs.

**Review by majecty — APPROVED** [review]

**Pull request merged**